api/src/main/java/jakarta/servlet/http/HttpSessionContext.java

```diff
@@ -55,5 +56,5 @@
      *
      */
     @Deprecated
-    public Enumeration<String> getIds();
+    default public Enumeration<String> getIds() {return null; }
```

Comment:
java.util.Collections#emptyEnumeration() ?

The deprecation message says `This method must return an empty <code>Enumeration</code>`

Reply:
It also says `@return null`